common/util/file_util.cc

```diff
@@ -126,9 +126,18 @@ absl::Status UpwardFileSearch(absl::string_view start,
   return absl::NotFoundError("No matching file found.");
 }
 
+absl::Status FileExists(const std::string &filename) {
+  struct stat file_info;
+  if (stat(filename.c_str(), &file_info) == 0) {
```

Comment:
This just means that `stat()` call was OK. Not necessary that this is a file.

You need to explicitly check if that's a file or not: https://linux.die.net/man/2/stat (you can use `S_ISREG` macro). Here's an example: https://cs.opensource.google/verible/verible/+/master:common/util/file_util.cc;drc=75c8ea36a5ef1e1b642660b0555ea9fb086d8271;l=210